apps/archive/autocomplete.py

```diff
@@ -0,0 +1,53 @@
+
+import locale
+import superdesk
+
+from flask import current_app as app, request
+from datetime import timedelta
+from superdesk.utils import ListCursor
+from superdesk.utc import utcnow
+from superdesk.errors import SuperdeskApiError
+
+
+class AutocompleteResource(superdesk.Resource):
+    item_methods = []
+    resource_methods = ['GET']
+
+
+class AutocompleteService(superdesk.Service):
+
+    allowed_fields = {
+        'slugline',
```

Comment:
Being as this endpoint is quite generic, what if we were to allow any non-system level fields and string only fields.

For example:
slugline
headline
abstract
ednote
description_text
archive_description
alt_text

Reply:
I think for those non single line fields it could be a lot of data, also not sure how the autocomplete would work there in the editor. but I'm open to add more if needed.